source/interact-data/crud.txt

```diff
@@ -0,0 +1,268 @@
+    To learn more about Django's ``QuerySet`` API, see
+    `QuerySet API reference <https://docs.djangoproject.com/en/5.1/ref/models/querysets/>`__
+    in the Django documentation.
+
+You can also use the Django admin site to edit your models
+and their corresponding collections on a web interface. For
+more information, see `The Django admin site <https://docs.djangoproject.com/en/5.1/ref/contrib/admin/>`__
+in the Django documentation.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``Movie`` model, which represents
+the ``sample_mflix.movies`` collection from the :atlas:`Atlas sample datasets </sample-data>`.
+The ``Movie`` model class has the following definition:
+
+.. code-block:: python
+
+    from django.db import models
+    from django_mongodb_backend.fields import EmbeddedModelField, ArrayField
+
+   class Movie(models.Model):
+       title = models.CharField(max_length=200)
+       plot = models.TextField(blank=True)
+       runtime = models.IntegerField(default=0)
+       released = models.DateTimeField("release date", null=True, blank=True)
+       awards = EmbeddedModelField(Award)
+       genres = ArrayField(models.CharField(max_length=100), null=True, blank=True)
+
+       class Meta:
+           db_table = "movies"
+           managed = False
+      
+       def __str__(self):
+           return self.title
+
+You can use the Python interactive shell to run the code examples.
+To enter the shell, run the following command from your project's 
+root directory:
+
+.. code-block:: bash
+
+   python manage.py shell
+
+After entering the Python shell, ensure that you import the following models and
+modules:
+
+.. code-block:: python
+
+   from <your application name>.models import Movie
+   from django.utils import timezone
+   from datetime import datetime
+
+To learn how to create a Django application that uses the ``Movie``
+model and the Python interactive shell to interact with MongoDB documents,
+visit the :ref:`django-get-started` tutorial.
+
+.. _django-crud-insert:
+
+Insert Documents
+----------------
+
+To insert a document into a collection, call the ``create()`` method
+on an instance of your model's ``Manager`` class. Pass the new document's
+fields and values as arguments to the ``create()`` method.
+
+Example
+~~~~~~~
+
+The following example calls the ``create()`` method to insert a document
+into the ``sample_mflix.movies`` collection. The new document has
+a ``title`` value of ``"Poor Things"`` and a ``runtime`` value
+of ``141``:
+
+.. code-block:: python
+
+   movie = Movie.objects.create(title="Poor Things", runtime=141)
+
+.. note::
+
+   The ``create()`` method allows you to create a new ``Movie`` object
+   and save the object as a collection document in one method call.
+   To view an example that creates an object then saves it to the
+   database by calling ``save()``, see `create() <https://docs.djangoproject.com/en/5.1/ref/models/querysets/#create>`__
+   in the Django documentation.
```

Comment:
I don't know if you want to mention this shortcut here, but I think you can also do this:

```
movie = Movie(title="Poor Things", runtime=141)
```

Presumably this works because instantiation of the class calls the manager's `create` method or something like that.

Reply:
Looks like you still need to call save() after to actually insert it into the database